tests/test_qiskit_device.py

```diff
@@ -173,7 +191,7 @@ def test_no_warning_raised_for_software_backend_analytic_expval(
         """Tests that no warning is raised if the analytic attribute is true on
         statevector simulators when calculating the expectation"""
 
-        dev = qml.device("qiskit.aer", backend="aer_simulator", method=method, wires=2, shots=None)
+        _ = qml.device("qiskit.aer", backend="aer_simulator", method=method, wires=2, shots=None)
```

Comment:
Might do something similar to this above.

Reply:
I am doing this assignment here since the statement occurred without a `with` context.